app/src/main/kotlin/com/wire/android/ui/calling/SharedCallingViewModel.kt

```diff
@@ -238,8 +238,10 @@ class SharedCallingViewModel @Inject constructor(
         viewModelScope.launch {
             callState.isMuted?.let {
                 callState = if (it) {
+                    unMuteCall(conversationId)
```

Comment:
Wasn't there a reason we remove the usecase invokation from here?

I think it was because while we were in an ongoing call and received another call, our microphone state in the ongoing call would change 🤔

Reply:
yes the issue with changing microphone state of incoming call when your are already on a call. 
But it should be fine now since we are checking before calling AVS muting method that the call is established.

That check was added in older PR